firebase_admin/messaging.py

```diff
@@ -622,6 +636,7 @@ def encode_webpush(cls, webpush):
             'headers': _Validators.check_string_dict(
                 'WebpushConfig.headers', webpush.headers),
             'notification': cls.encode_webpush_notification(webpush.notification),
+            'fcm_options': cls.encode_webpush_fcm_options(webpush.fcm_options)
```

Comment:
The json key should be called `fcmOptions`. The REST API documentation mentions `fcm_options` by error and the FCM team is going to fix it.